src/NzbDrone.Core/Music/TrackRepository.cs

```diff
@@ -137,6 +138,21 @@ public List<Track> TracksWithFiles(int artistId)
             return Query.QueryText(query).ToList();
         }
 
+        public List<Track> TracksWithoutFiles(int albumId)
+        {
+            string query = string.Format("SELECT Tracks.* " +
```

Comment:
Is calling this extra query with multiple joins necessary here, I would think we would see some degradation of speed on release processing calling this on every release

Reply:
I guess this is a bit pointless calling for every release since the existence of missing tracks will be the same for every release.  Not sure of the best way of avoiding the repetitive calls in the existing framework though.